<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewPartDialog</class>
 <widget class="QDialog" name="NewPartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neues Bauteil eintragen</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>593</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="grp_BaseInformation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Basisinformationen</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>32</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl_ImagePreview">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <property name="text">
        <string>IMAGE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ChooseImage">
         <property name="text">
          <string>Bild w√§hlen</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>32</y>
      <width>245</width>
      <height>240</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_Name">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edt_PartName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_ShortDescription">
       <property name="text">
        <string>Kurzbeschreibung</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edt_ShortDescription"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_Type">
       <property name="text">
        <string>Typ</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="edt_Type"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_Category">
       <property name="text">
        <string>Kategorie</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="edt_Category"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbl_Subcategory">
       <property name="text">
        <string>Unterkategorie</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="edt_Subcategory"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lbl_Format">
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="edt_Format"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lbl_Quantity">
       <property name="text">
        <string>Menge</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="spb_Quantity">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lbl_Price">
       <property name="text">
        <string>Preis</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="edt_Price"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_Links">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>349</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Links</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>20</y>
      <width>331</width>
      <height>193</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_Source">
       <property name="text">
        <string>Bezugsquelle</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbb_Source">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_Source_2">
       <property name="text">
        <string>Link Bezugsquelle </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edt_SourceLink"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_AlternativeSource">
       <property name="text">
        <string>Alternative Bezugsquelle</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cbb_AlternativeSource">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_Source_3">
       <property name="text">
        <string>Link Bezugsquelle </string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="edt_AlternativeSourceLink"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbl_Manufacturer">
       <property name="text">
        <string>Hersteller</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="cbb_Manufacturer">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lbl_Manufacturer_2">
       <property name="text">
        <string>Link Hersteller</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="edt_ManufacturerLink"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>871</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Beschreibung</string>
   </property>
   <widget class="QTextEdit" name="txt_Description">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>851</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_StorageLocation">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>240</y>
     <width>351</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Lagerort</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>26</y>
      <width>331</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_StorageLocation">
       <property name="text">
        <string>Lagerort                           </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edt_StorageLocation"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edt_StorageLocationDetails"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_StorageLocationDetails">
       <property name="text">
        <string>Details</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_Files">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>511</width>
     <height>70</height>
    </rect>
   </property>
   <property name="title">
    <string>Dateien / Bilder</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>491</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbl_FilePath">
       <property name="text">
        <string>Basispfad</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edt_PartFilesFolder"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_SelectPartFilesFolder">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewPartDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewPartDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
